resources/views/layouts/app.blade.php

```diff
@@ -25,7 +25,7 @@
     @stack('head')
     @config('design/head/includes')
 </head>
-<body class="text-neutral antialiased">
+<body class="text-neutral antialiased has-[.prevent-scroll]:max-md:overflow-clip">
```

Comment:
why add overflow-clip only on mobile? this allows a body scroll on desktop when a slideover is opened?